src/Components/ClusterRules/ClusterRules.cy.js

```diff
@@ -122,6 +122,9 @@ describe('test data', () => {
     expect(filteredData).to.have.length.gte(1);
     expect(filteredData).to.have.length.lt(RULES_ENABLED);
   });
+  it('has 1 rule without impacted field', () => {
```

Comment:
I feel this too much restricting. I would test for something like, `'at least 1 rule without impacted field'`. The reason is that a new entry without impacted field will make this test to fail, but the tests below do not require one and only one, but any number of rules without impacted should work.